packages/block-library/src/footnotes/index.php

```diff
@@ -208,7 +208,7 @@ function wp_add_footnotes_to_revision( $fields ) {
  * @param object $revision       The revision object to compare against.
  * @return string The field value.
  */
-function wp_get_footnotes_from_revision( $revision_field, $field, $revision ) {
+function _wp_get_footnotes_from_revision( $revision_field, $field, $revision ) {
 	return get_metadata( 'post', $revision->ID, $field, true );
 }
-add_filter( 'wp_post_revision_field_footnotes', 'wp_get_footnotes_from_revision', 10, 3 );
+add_filter( 'wp_post_revision_field_footnotes', '_wp_get_footnotes_from_revision', 10, 3 );
```

Comment:
I believe this hook name is incorrect here and needs a `_` prefix, not the callbacks.

See: https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/revision.php#L90-L93

Reply:
Look like I reverted a part of the earlier PR by accident.

Reply:
Added it back. Still doesn't explain to me why it was working fine previously.

Reply:
I'm not sure, either.

Do you mind reverting callback renames? Those are unnecessary.

Reply:
Sure. Are they completely unnecessary? We don't want people to be using these functions.

Reply:
Usually underscore prefix indicates private functions, but there's no such thing as private global functions in PHP. If people want, they'll still use it.

Using closures was a good idea, but I don't fully understand why it was requested to revert that.